gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileSystemProvider.java

```diff
@@ -66,14 +66,14 @@
   private final Storage storage;
 
   // used only when we create a new instance of CloudStorageFileSystemProvider.
-  private static StorageOptions storageOptions;
+  private static StorageOptions defaultStorageOptions;
 
   /**
-   * Sets options that are only used by the constructor.
+   * Sets default options that are only used by the constructor.
    */
   @VisibleForTesting
   public static void setGCloudOptions(StorageOptions newStorageOptions) {
-    storageOptions = newStorageOptions;
+    defaultStorageOptions = newStorageOptions;
```

Comment:
No, but since this code will be completely changed by @jart on her next PR it'll be up to her to consider that option.